state-network.md

```diff
@@ -348,21 +354,21 @@ root, with the final round of gossip only containing the `[A]` which is the
 state root node of the trie.
 
 
-### Gossip 
-
-> TODO: this section still needs to be updated.
+### Bridge Node Responsibilities
 
-Each time a new block is added to their view of the chain, a set of merkle proofs which are all anchored to `Header.state_root` is generated which contains:
+Each time a new block is added to their view of the chain, a set of merkle
+proofs which are all anchored to `Header.state_root` is generated which
+contains:
 
 - Account trie Data:
+    - All of the new and modified account nodes from the state trie.
     - All of the intermediate and leaf trie nodes from the account trie necessary to prove new and modified accounts.
 - Contract Storage trie data:
+    - All of the new and modified storage slots from each modified contract storage trie.
     - All of the intermediate and leaf trie nodes from each contract storage trie necessary to prove new and modified storage slots.
 - All contract bytecode for newly created contracts
 
-> TODO: Figure out language for defining which trie nodes from this proof the bridge node must initialize gossip.
-
-> TODO: Determine mechanism for contract code.
-
-The receiving DHT node will propagate the data to nearby nodes from their routing table.
-
+A bridge should compute the content-id values for all of this new state data
+and sort the data by proximity to its own node-id.  Beginning with the content
+that is *closest* to its own node-id it should proceed to GOSSIP each
```

Comment:
Should the content ordering by the bridge only apply to the leaves of account trie, storage trie, and contract code, considering the intermediate trie nodes will be gossiped by other nodes in the recursive gossip?

Reply:
I can re-word to make this explicit.